doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -898,8 +906,9 @@ The locator tags can use different wireless technologies like GPS, Bluetooth LE
 It is even possible to combine multiple technologies in a single product to improve the user experience.
 
 The `Fast Pair Device Feature Requirements for Locator Tags`_ documentation defines the Fast Pair requirements for the locator tag use case.
-If your product is targeting the locator tag use case, you must configure your application according to these requirements.
-Enable the mandatory Fast Pair features and extensions using the appropriate Kconfig options in your application's configuration.
+If your product is targeting the locator tag use case, you must manually configure your application according to these requirements in your project configuration file.
```

Comment:
I would not inform about possibility of manual configuration. I think we should suggest only to rely on `CONFIG_BT_FAST_PAIR_USE_CASE_LOCATOR_TAG` to simplify doc here